src/pseudopeople/noise.py

```diff
@@ -33,20 +33,28 @@ def noise_form(
     randomness = get_randomness_stream(form, seed)
 
     for noise_type in NOISE_TYPES:
-        noise_configuration = configuration[form][noise_type]
+        noise_configuration = configuration[form.value]
         if isinstance(noise_type, RowNoiseType):
             # Apply row noise
-            form_data = noise_type(form_data, noise_configuration, randomness)
+            print(noise_type.name)
+            form_data = noise_type(
+                form_data, noise_configuration, randomness, noise_type.name
+            )
 
         elif isinstance(noise_type, ColumnNoiseType):
+            columns_to_noise = [
```

Comment:
You don't need to use`config.keys()` or `config.to_dict()`. `__contains__()` is implemented on `ConfigTree`, so you can just use python `in` syntax like we had had before.